docs/src/app/(public)/(content)/react/utils/use-render/demos/render/css-modules/index.module.css

```diff
@@ -0,0 +1,5 @@
+.Text {
+  font-size: 0.875rem;
+  line-height: 1rem;
+  color: var(--color-gray-900);
+}
```

Comment:
`strong` has `font-weight: 700` by default, but it looks ugly. I'd use `500` or `600` instead.


```suggestion
  strong& {
    font-weight: 500;
  }
}
```

|before|after|
|---|---|
|![deploy-preview-1418--base-ui netlify app_react_utils_use-render (3)](https://github.com/user-attachments/assets/1cc87d54-5edf-40ef-958c-31625463e78f)|![localhost_3005_react_utils_use-render](https://github.com/user-attachments/assets/7318378d-8a9e-49cb-9138-ce7fe4ba3cd0)|